packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -641,11 +660,17 @@ export class SmoothControls extends EventDispatcher {
         scene.getNDC(pointer.clientX, pointer.clientY));
 
     if (hit == null) {
-      const {cameraTarget} = scene.element;
-      scene.element.cameraTarget = '';
-      scene.element.cameraTarget = cameraTarget;
-      // Zoom all the way out.
-      this.userAdjustOrbit(0, 0, 1);
+      // 'This should probably be optional, but I don't know how to extend it to pass something into here
+
+      // No, don't move all the way out.
+      // If clicked on nothing then do nothing.
+      // Resetting on hit == null means you forfeit the position you were just in. If that position is intentional then you lose that intention.
```

Comment:
Can you describe the situations where you find this to be a problem? This behavior was put in to allow the user to return to the initial pan state, which is basically impossible when panning back and forth.

Reply:
Thank you for the explanation! That makes sense, but I'd rather fix the bug regarding incorrect zoom-out. You're absolutely right that it should be returning you to your initial view (with the exception of leaving theta and phi at their current values). It would probably be a good idea to add an example with this extreme FoV so we can be sure it's fixed and doesn't regress. It's also possible a small configuration change will help, if I can see what you're doing.

Yes, if you want selection, then single-click is nice for that, but we don't do selection in MV. Double-click we've found to be just too hard to discover, especially when single click does nothing.